Remove decimal places from battery number

pmahnke opened the pull request:
The current command removes numbers in the format `##,###`; however, my computer's output is `##.###` so I added an additional `cut` command to remove both types of decimals. Note, it doesn't round.

[before]
![image](https://user-images.githubusercontent.com/441217/96774850-90ec3a00-13de-11eb-8c7b-e6fb784a0912.png)

[after]
![image](https://user-images.githubusercontent.com/441217/96774991-c2650580-13de-11eb-915b-4f34ab81df0b.png)

moritzheiber commented:
Hey, I think the same can be achieved by changing `print $2` in the `awk` script to `print int($2)`. Could you confirm? And if it does solve the issue, could you modify your PR accordingly?

To be clear, with casting the number to int, both `cut` statements should be superfluous :slightly_smiling_face: 

I can't test it myself, because the `upower` line for me reports relative percentages (e.g `99%`).

PS: Are you testing against the latest version of the script? Because the line generating `$__UPOWER_INFO` has changed recently.

pmahnke commented on scripts/battery:
```diff
@@ -19,7 +19,7 @@ get_battery_status() {
 #   The battery's percentage, without the %.
 get_battery_percent() {
-    echo "$__UPOWER_INFO" | awk -W posix '$1 == "percentage:" { gsub("%","",$2); print $2}' | cut -d',' -f1
+    echo "$__UPOWER_INFO" | awk -W posix '$1 == "percentage:" { gsub("%","",$2); print $2}' | cut -d',' -f1 | cut -d'.' -f1 
```
```suggestion
    echo "$__UPOWER_INFO" | awk -W posix '$1 == "percentage:" { gsub("%","",$2); print int($2)}'
```

pmahnke commented:
@moritzheiber I can confirm it worked for me and updated my PR.

I am not sure I am using the correct UPOWER script, but I did modify the battery script directly to test.

moritzheiber commented:
Great, thanks!

kgilmer commented:
Like @moritzheiber , I have never seen a non whole number in the battery status indicator.  @pmahnke have you always had this?  Any idea why?

pmahnke commented:
@kgilmer I assume it was part of PR #77 when the script moved to use `DisplayDevice` in the `upower` command.  I assume that command does some averaging that creates the float.

If you see the output of upower for `DisplayDevice` it is a float where BAT0 and BAT1 are integers.  The person who made that PR tried to remove the decimals, but it was for only the countries that use a comma as a delimiter, not a general solution.

I hope that makes sense?

[output from my laptop with 2 batteries]
```
 ⚙  ~  upower --show-info "/org/freedesktop/UPower/devices/battery_BAT0" 
  native-path:          BAT0
  vendor:               SMP
  model:                01AV421
  serial:               10682
  power supply:         yes
  updated:              Fri 23 Oct 2020 08:19:04 BST (118 seconds ago)
  has history:          yes
  has statistics:       yes
  battery
    present:             yes
    rechargeable:        yes
    state:               fully-charged
    warning-level:       none
    energy:              19.43 Wh
    energy-empty:        0 Wh
    energy-full:         19.43 Wh
    energy-full-design:  24 Wh
    energy-rate:         0 W
    voltage:             12.601 V
    percentage:          100%
    capacity:            80.9583%
    technology:          lithium-polymer
    icon-name:          'battery-full-charged-symbolic'

 ⚙  ~  upower --show-info "/org/freedesktop/UPower/devices/battery_BAT1"
  native-path:          BAT1
  vendor:               LGC
  model:                01AV492
  serial:               512
  power supply:         yes
  updated:              Fri 23 Oct 2020 08:21:04 BST (4 seconds ago)
  has history:          yes
  has statistics:       yes
  battery
    present:             yes
    rechargeable:        yes
    state:               fully-charged
    warning-level:       none
    energy:              62.94 Wh
    energy-empty:        0 Wh
    energy-full:         63.44 Wh
    energy-full-design:  71.1 Wh
    energy-rate:         10.866 W
    voltage:             12.818 V
    percentage:          99%
    capacity:            88.9451%
    technology:          lithium-ion
    icon-name:          'battery-full-charged-symbolic'

 ⚙  ~  upower --show-info "/org/freedesktop/UPower/devices/DisplayDevice"
  power supply:         yes
  updated:              Fri 23 Oct 2020 08:15:02 BST (369 seconds ago)
  has history:          no
  has statistics:       no
  battery
    present:             yes
    state:               fully-charged
    warning-level:       none
    energy:              82.37 Wh
    energy-full:         82.87 Wh
    energy-rate:         10.866 W
    percentage:          99.3966%
    icon-name:          'battery-full-charged-symbolic'
```

kgilmer commented:
That makes perfect sense, thank you for the detailed response @pmahnke . :smile:   I have only one battery so I did not see the bug.
